src/index.js

```diff
@@ -3,10 +3,17 @@ import ReactDOM from "react-dom";
 
 import "./styles.css";
 import { Game } from "./Game";
+import { BOARD_SIZE_MAX, BOARD_SIZE_MIN } from './constants';
 
-const App = function() {
-  const boardSizeX = 10;
-  const boardSizeY = 10;
+const App = function () {
+  // eslint-disable-next-line
```

Comment:
what was the error for which eslint raised error?

Reply:
Unexpected use of 'location'  no-restricted-globals